src/DependencyInjection/SchedulerBundleExtension.php

```diff
@@ -561,7 +569,8 @@ private function registerBuilders(ContainerBuilder $container): void
         $chainedBuilderDefinition = new ChildDefinition(AbstractTaskBuilder::class);
         $chainedBuilderDefinition->setClass(ChainedBuilder::class);
         $container->setDefinition(ChainedBuilder::class, $chainedBuilderDefinition)
-            ->setArgument(1, new TaggedIteratorArgument('scheduler.task_builder'))
+            ->setArgument(1, new Reference(SchedulePolicyOrchestratorInterface::class, ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE), )
+            ->setArgument(2, new TaggedIteratorArgument('scheduler.task_builder'))
```

Comment:
Regarding the fact that we improve the actual definition, I prefer to let the `TaggedIteratorArgument` as the "first" argument and add the orchestrator as the second one.

Reply:
I agree with that, but if I set orchestrator as second one then I got an `optional parameter is provided before required one` :(

Reply:
Yes, my bad on this one, I've forgot that the builders are an optional one, solved 🙂